src/lib/genesis_constants/genesis_constants.ml

```diff
@@ -41,6 +57,78 @@ module Constraint_constants = struct
     }
   [@@deriving bin_io_unversioned, sexp, equal, compare, yojson]
 
+  let make (inputs : Inputs.t) : t =
+    (** All the proofs before the last [work_delay] blocks must be
```

Comment:
Is this a documentation for the `work_delay` flag in particular?.. Or is it also related to the computations below?